package.json

```diff
@@ -20,6 +20,6 @@
   "dependencies": {
     "gl-matrix": "^3.1.0",
     "three": "^0.111.0",
-    "webxr-polyfill": "^2.0.3"
+    "webxr-polyfill": "git+https://github.com/fe1ixz/webxr-polyfill.git"
```

Comment:
Would you please update the dependency to the official one because the PR https://github.com/immersive-web/webxr-polyfill/pull/163 has been merged?

Reply:
> 

Hi @takahirox ,

Sorry for the late update, planning at work for the new year has kept me busy for quite some time. Immersive web has merged the webxr-polyfill change but has not published a new version in npm, as such, I modified the dependency from my git repo to theirs. Is this okay for now? If you think so I shall start preparing for the next PR for the emulator.

Reply:
Yes, that sounds good to me. Thanks!